tests/test_transports.py

```diff
@@ -0,0 +1,30 @@
+import time
+import pytest
+import urllib3
+from epsagon.trace_transports import HTTPTransport
+from epsagon.trace import (trace_factory)
+
+
+def test_sanity(httpserver):
```

Comment:
would add test_httptransport_sanity (the same for the timeout test)

Reply:
Done